net_udp.go

```diff
@@ -54,9 +55,9 @@ type (
 	// For the proc file format details, see https://linux.die.net/man/5/proc.
 	netUDPLine struct {
 		Sl        uint64
-		LocalAddr uint64
+		LocalAddr []uint8
```

Comment:
Maybe we should use Go [net.IP](https://golang.org/pkg/net/#IP) here?

Reply:
@SuperQ Done.